vars/checkPatch.groovy

```diff
@@ -1,13 +1,33 @@
 // vars/checkPatch.groovy
 
+/**
+ * checkPatch.groovy
+ *
+ * checkPatch variable
+ */
+
 import com.intel.checkoutScm
 
-def call(Map config) {
+/**
+ * Method to check the status of a patch and notify GitHub of the results
+ * 
+ * @param config Map of parameters passed.
+ *
+ * config['jenkins_review'] is the optional revision number to use.
+ *
+ */
+def call(Map config = [:]) {
+    if (env.DAOS_JENKINS_NOTIFY_STATUS == null) {
```

Comment:
So we should set the global property before we land this, yes?

Reply:
Yes on Jenkins-3 for now and then switch it for production.
I think that all build agents need a restart to pick up the change.

Reply:
I've added that to _jenkins-3_.